src/evaluate.py

```diff
@@ -38,7 +40,7 @@ def parse_write_preds_arg(write_preds_arg: str) -> List[str]:
 def evaluate(model, tasks: Sequence[tasks_module.Task], batch_size: int,
              cuda_device: int, split="val") -> Tuple[Dict, pd.DataFrame]:
     '''Evaluate on a dataset'''
-    FIELDS_TO_EXPORT = ['idx', 'sent1_str', 'sent2_str', 'labels']
+    FIELDS_TO_EXPORT = ['idx', 'sent1_str', 'sent2_str', 'labels', 'qst_idx', 'ans_idx']
```

Comment:
I assume you've confirmed that this doesn't break other tasks.

Reply:
Yea, there's a line shortly below that's `for field in FIELDS_TO_EXPORT: if field in batch`